src/components/movies/MovieCard.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <div class="movie__card" v-if="movie">
+        <h1 class="movie__title"> {{ movieTitle }}</h1>
+        <div class="movie__info">
+            <p class="movie__duration-time">{{ movieLength }}</p>
+            <img class="movie__poster" :src="moviePoster" />
+            <div class="movie__category">
+                {{ movieCategory }}
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    props: {
+        movie: {
+            type: Object
```

Comment:
it is a good practice to have prop definitions as detailed as possible. Especially to always have a `type` definition (like you do 👍) and `default` value specified (if a prop is not `required`)